### ISSUE TITLE
Reasoner-validator install broken

### ISSUE THREAD

**Issue by finnagin** [RTXteam/RTX#1490]
https://travis-ci.com/github/RTXteam/RTX/builds/227407827

The install command for reasoner-validator is now breaking in travis now:
```
$ pip install -r requirements.txt

Looking in indexes: https://pypi.python.org/simple/

Collecting git+git://github.com/NCATSTranslator/reasoner-validator@2.0 (from -r requirements.txt (line 33))

  Cloning git://github.com/NCATSTranslator/reasoner-validator (to revision 2.0) to /tmp/pip-req-build-f0csrv6u

  WARNING: Did not find branch or tag '2.0', assuming revision or ref.

ERROR: Command errored out with exit status 1: git checkout -q 2.0 Check the logs for full command output.
```

 It looks like it was updated to 2.0.0 in [pypi](https://pypi.org/project/reasoner-validator/) so maybe we can install with just a install form there with `resoner-validator==2.0.0`?

**Comment by finnagin** [comment]
@edeutsch I'm going to try to fix this lmk if I should switch it back for any reason.

**Comment by finnagin** [comment]
Looks like that worked: https://travis-ci.com/github/RTXteam/RTX/builds/227471860

**Comment by finnagin** [comment]
I think this one is good to close unless there is some specific github branch we should be installing from rather than the pypi version.

**Comment by edeutsch** [comment]
Thanks for handling this while I was away, Finn! I think this is all fine. I understand this all came about because Patrick merged branch "2.0" in the validator and then deleted the branch and then pushed it to PyPI. Otherwise it is the same code.
Thanks! Closing.
